src/Application/Controller/Project/ProjectController.php

```diff
@@ -418,6 +418,29 @@ public function projectCommentDetail(string $id): Response
     ]);
   }
 
+  #[Route(path: '/markNotForKids/{id}', name: 'mark_not_for_kids', methods: ['POST'])]
+  public function markNotForKids(string $id): Response
+  {
+    $project = $this->project_manager->find($id);
+    if (null === $project) {
+      return $this->redirectToIndexOnError();
+    }
+    if (2 == $project->getNotForKids()) {
```

Comment:
```suggestion
    if (<const>== $project->getNotForKids()) {
```

Use constants please